internal/rootcoord/import_manager.go

```diff
@@ -729,3 +729,18 @@ func taskExpired(ti *datapb.ImportTaskInfo) bool {
 		ti.GetState().GetStateCode() != commonpb.ImportState_ImportCompleted &&
 		Params.RootCoordCfg.ImportTaskExpiration <= float64(time.Now().Unix()-ti.GetCreateTs())
 }
+
+func (m *importManager) GetImportFailedSegmentIDs() ([]int64, error) {
+	ret := make([]int64, 0, 10)
```

Comment:
Why is there a cap of 10 and what happens if it is exceeded?

Reply:
If it's exceeded, the slice will be copy to a new memory space and enlarge its capacity.

Reply:
If we not going to limit the size, we can just do: `ret := make([]int64, 0)`.

Reply:
ok